### DISCUSSION TITLE
The ASCII art of 'LAZYVIM' on the LazyVim entry page is not displaying.

### DISCUSSION THREAD

**Discussion by coorus** [LazyVim/LazyVim#4322] [Q&A]
Could you please help me troubleshoot this issue? I have downloaded and am using the JetBrains Mono Nerd Font, but the part of the Neovim entry page that should display 'LAZYVIM' is missing.
<img width="1440" alt="截屏2024-08-19 05 19 37" src="https://github.com/user-attachments/assets/5af70df2-6f9e-4779-ae90-c9a1039791ae">

**Comment by dpetka2001** [comment]
Must be your terminal. Make sure you have set it up to use the fonts you installed, but I've read about different discrepancies with iTerm (but I don't have experience with Mac myself, so can't be sure 100%). Kitty or Wezterm usually behave better if you'd like to give them a try.

**Reply by coorus** [reply]
Thank you so much for your reply! 😊 I was indeed facing this issue while using iTerm2. Following your advice, I’ve now successfully tried Tabby, Kitty, Warp, and Wezterm, and they all display correctly. It looks like it’s a GPU rendering issue with iTerm2. Thanks a ton! 🙌🎉
![QQ20240819-155621](https://github.com/user-attachments/assets/f7131ce2-e851-468d-baaa-f894d66d2c1e)
